tests/api/v1/datadog/cassettes/TestServicesLifecycle.freeze

```diff
@@ -0,0 +1 @@
+2020-03-09T12:04:46.140867-04:00
```

Comment:
I noticed that these teams and services cassettes are using `v1/`. These can probably be deleted

Reply:
Good catch, those must have been leftovers from the last pr